packages/compass-import-export/src/import/import-csv.spec.ts

```diff
@@ -242,6 +244,8 @@ describe('importCSV', function () {
 
         const output = temp.createWriteStream();
 
+        debug('output path', output.path);
```

Comment:
So we can confirm that the path we see in the ENOENT error is actually the temp output path.